qddssrc/nemp.dspf

```diff
@@ -6,9 +6,9 @@
      A                                      DSPATR(UL)
      A            XID            6A  O  6 14
 
-     A                                  7  7'First'
+     A                                  7  7'F.'
```

Comment:
Not sure why you are making it more cryptic, but I assume this is just from demo purposes